doc/articles/contributing/guidelines/testing.md

```diff
@@ -0,0 +1,81 @@
+# Guidelines for authoring tests
+
+
+## integration testing
+
+### Concepts
+
+- `Constants.cs`
+- `Run(xxxx)` specifics the full class path to the control
+- `SamplesApp*`
+- https://github.com/unoplatform/Uno.UITest
+
+### Dev Loop
+
+1. Start emulators and deploy application (iOS and Android)
+1. Start without debugging to ensure the webserver is running (webasm)
```

Comment:
```suggestion
1. Start without debugging to ensure the webserver is running (Wasm)
```

It's either _WebAssembly_ or _Wasm_ (this specific casing and spaces): See https://webassembly.org: 
![image](https://user-images.githubusercontent.com/5839577/66649009-05e93900-ebfb-11e9-933d-3478b51c3f3f.png)